tests/configuration_manager_test.go

```diff
@@ -110,4 +115,16 @@ func TestGetConfigFromEnvVariables(t *testing.T) {
 	if options.FailOverStrategy != enums.FailOverStrategy.FailClose {
 		t.Errorf("Test Failed: expected to reiecve: %s, got: %s", enums.FailOverStrategy.FailClose, options.FailOverStrategy)
 	}
+
+	if len(options.ProxyHeaders) != len([]string{"CF-Connecting-Ip", "Some-Random-Ip"}) {
```

Comment:
I know it's a test but better to extract en([]string{"CF-Connecting-Ip", "Some-Random-Ip"} abpve and not to do in if